testing/snapshot_test.ts

```diff
@@ -459,6 +478,148 @@ Deno.test(
   }),
 );
 
+Deno.test(
+  "Snapshot Test - Different Dir",
+  testFnWithDifferentTempDir(async (t, tempDir1, tempDir2) => {
+    const tempTestFileName = "test.ts";
+    const tempTestFilePath1 = join(tempDir1, tempTestFileName);
+    const tempSnapshotFilePath1 = join(
+      tempDir1,
+      "__snapshots__",
+      `${tempTestFileName}.snap`,
+    );
+    const tempTestFilePath2 = join(tempDir2, tempTestFileName);
+    const tempSnapshotFilePath2 = join(
+      tempDir2,
+      "__snapshots__",
+      `${tempTestFileName}.snap`,
+    );
+
+    async function runTestWithUpdateFlag(test1: string, test2: string) {
+      await Deno.writeTextFile(tempTestFilePath1, test1);
+      await Deno.writeTextFile(tempTestFilePath2, test2);
+
+      const command = new Deno.Command(Deno.execPath(), {
+        args: [
+          "test",
+          "--allow-all",
+          tempTestFilePath1,
+          tempTestFilePath2,
+          "--",
+          "-u",
+        ],
+      });
+      const { stdout, stderr } = await command.output();
+
+      return {
+        output: new TextDecoder().decode(stdout),
+        error: new TextDecoder().decode(stderr),
+        snapshots1: await Deno.readTextFile(tempSnapshotFilePath1),
+        snapshots2: await Deno.readTextFile(tempSnapshotFilePath2),
+      };
+    }
+
+    function assertNoError(error: string) {
+      if (formatTestError(error)) {
+        throw new AssertionError(`Unexpected Error:\n\n${error}\n`);
+      }
+    }
+
+    /**
+     * New snapshot
+     */
+    const result1 = await runTestWithUpdateFlag(
+      `
+      import { assertSnapshot } from "${SNAPSHOT_MODULE_URL}";
+
+      Deno.test("Snapshot Test - First", async (t) => {
+        await assertSnapshot(t, [
+          1,
+          2,
+        ]);
+      });
+      Deno.test("Snapshot Test - Second", async (t) => {
+        await assertSnapshot(t, [
+          3,
+          4,
+        ]);
+      });`,
+      `
+      import { assertSnapshot } from "${SNAPSHOT_MODULE_URL}";
+
+      Deno.test("Snapshot Test - First", async (t) => {
+        await assertSnapshot(t, [
+          1,
+          2,
+        ]);
+      });
+      Deno.test("Snapshot Test - Second", async (t) => {
+        await assertSnapshot(t, [
+          3,
+          4,
+        ]);
+      });`,
+    );
+
+    assertNoError(result1.error);
+    await assertSnapshot(t, formatTestOutput(result1.output), {
+      name: "Snapshot Test - Different Dir - New snapshot",
+    });
```

Comment:
I think this assertion is enough for checking this change isn't breaking things. Can you remove the rest of assertions and reduce the new snapshots?

Reply:
removed https://github.com/denoland/deno_std/pull/3422/commits/e7853d18390f880d036dafe77d07f2ee27c07b15